### DISCUSSION TITLE
UBL with BLTouch probe versus regular with BLTouch probe?

### DISCUSSION THREAD

**Discussion by VashBaldeus** [Jyers/Marlin#533] [Q&A] [answered]
UBL with BLTouch probe versus regular with BLTouch probe?
What is the difference?

Also, according to wiki description of UBL it states there's (3x3, 5x5, 10x10, or 15x15) setups, but the lowest one is 10x10

**Comment by Jyers** [comment, marked as answer]
The wiki says it all. Primary benefit is the autotilt feature and persistent mesh storage. The mention of 3x3 and 5x5 are referring to the manual versions of UBL.
